cfpb/consumerfinance.gov
# Update look and feel of SBL FIG search tool

Applies various CSS classes to the FIG's search modal to make it look more like a cf.gov product.

See https://github.local/reg-tech/sbl-data-collection/issues/298

## Changes

- Brings in latest version of [ctrl-f](https://github.com/cfpb/ctrl-f) which includes a `Clear search` button to clear the search input.
- Applies our text input and icon classes to the search modal components.
- Some more tests.

## How to test this PR

(This branch has been deployed to dev4.)

1. Pull down this branch.
1. Functional tests should pass:
    `yarn cypress run --spec test/cypress/integration/compliance/small-business-lending/ --env FIG_URL=/data-research/small-business-lending/filing-instructions-guide/filing-instructions-guide-test/ --config baseUrl=https://secret-dev4-subdomain.cfpb.gov`
1. Manually check it out on [dev4](https://secret-dev4-subdomain.cfpb.gov/data-research/small-business-lending/filing-instructions-guide/202x-filing-instructions-guide/) if you so desire

## Screenshots

| before | after |
|--------|-------|
| <img width="982" alt="before" src="https://user-images.githubusercontent.com/1060248/206989046-9ba6e22a-d896-40f0-ae36-296cad9c7cc5.png">    | <img width="982" alt="after" src="https://user-images.githubusercontent.com/1060248/206989028-2bf8c611-e6ce-4db6-aaa1-aa7bc6aa781f.png">  |
